uhdm-tests/opentitan/Makefile.in

```diff
@@ -0,0 +1,123 @@
+curr_dir:=$(dir $(lastword $(MAKEFILE_LIST)))
+OPEN_TITAN = ${curr_dir}/opentitan
+
+################################
+#### verilator - simulation ####
+################################
+OPEN_TITAN_BUILD = ${root_dir}/build/lowrisc_systems_top_earlgrey_verilator_0.1
+
+OPEN_TITAN_INCLUDE = \
+	-I$(OPEN_TITAN_BUILD)/src/lowrisc_prim_assert_0.1/rtl \
+	-I$(OPEN_TITAN_BUILD)/src/lowrisc_ibex_ibex_core_0.1/rtl \
+	-I$(OPEN_TITAN_BUILD)/src/lowrisc_dv_dv_macros_0 \
+	-I$(OPEN_TITAN_BUILD)/src/lowrisc_prim_util_memload_0/rtl
+
+# All Ibex Core files that are needed
+#TO_SURELOG='ibex_core.sv\|ibex_pkg.sv\|prim_clock_gating.sv\|prim_generic_clock_gating.sv\|ibex_if_stage.sv\|ibex_prefetch_buffer.sv\|ibex_compressed_decoder.sv\|ibex_fetch_fifo.sv\|ibex_id_stage.sv\|ibex_decoder.sv\|ibex_controller.sv\|ibex_ex_block.sv\|ibex_alu.sv\|ibex_multdiv_fast.sv\|ibex_load_store_unit.sv\|ibex_wb_stage.sv\|ibex_cs_registers.sv\|ibex_register_file_ff.sv\|ibex_csr.sv\|ibex_counter.sv\|ibex_icache.sv\|ibex_pmp.sv\|prim_secded_28_22_enc.sv\|prim_secded_72_64_enc.sv\|prim_ram_1p.sv\|prim_secded_28_22_dec.sv\|prim_secded_72_64_dec.sv\|prim_generic_ram_1p.sv'
+
+# Working set of Ibex Core files
+TO_SURELOG='ibex_pkg.sv\|prim_generic_clock_gating.sv\|ibex_prefetch_buffer.sv\|ibex_compressed_decoder.sv\|ibex_fetch_fifo.sv\|ibex_id_stage.sv\|ibex_decoder.sv\|ibex_controller.sv\|ibex_ex_block.sv\|ibex_alu.sv\|ibex_multdiv_fast.sv\|ibex_load_store_unit.sv\|ibex_wb_stage.sv\|ibex_csr.sv\|ibex_register_file_ff.sv\|ibex_pmp.sv\|prim_secded_28_22_enc.sv\|prim_secded_72_64_enc.sv\|prim_secded_28_22_dec.sv\|prim_secded_72_64_dec.sv'
+
+# In order to parse full Ibex Core it is needed to add:
+# \|ibex_core.sv\|prim_clock_gating.sv\|ibex_cs_registers.sv\|ibex_counter.sv\|ibex_if_stage.sv\|ibex_icache.sv\|prim_ram_1p.sv\|prim_generic_ram_1p.sv
+
+# BUILD BRAKERS:
+# prim_generic_ram_1p.sv
+# ibex_counter.sv
+# prim_clock_gating.sv (silent error: %Error: Exiting due to 1 error(s))
+
+# prim_ram_1p.sv (prim_generic_ram_1p.sv fault)
+# ibex_icache.sv (requires prim_ram_1p.sv)
+# ibex_cs_registers.sv (ibex_counter.sv fault)
+# ibex_if_stage.sv (requires ibex_icache.sv)
+# ibex_core.sv (requires ibex_if_stage.sv ibex_cs_registers.sv prim_clock_gating.sv)
+
+SKIP_SOURCES='rv_core_ibex_pkg.sv'
```

Comment:
I think we can remove this variable by changing ``ibex_pkg.sv`` to ``/ibex_pkg.sv`` in ``TO_SURELOG`` variable.

Reply:
Fixed